### ISSUE TITLE
Add a parameter to preserve underscores when writing headers in LoadFromCollection<T> method

### ISSUE THREAD

**Issue by mamift** [EPPlusSoftware/EPPlus#91]
Hello. 

The default behaviour in the absence of a `DescriptionAttribute` or `DisplayNameAttribute` is to remove all underscores `_` and replace them with spaces, as evidenced at this line: https://github.com/EPPlusSoftware/EPPlus/blob/bc1ac1240e9b2cba1ed4057134e90dd8d108c5d1/src/EPPlus/ExcelRangeBase_Load.cs#L369

Can an option be added to preserve underscores? Preferably another parameter to the `LoadFromCollection<T>` method. Removing underscores is a sensible default, but I've come across a scenario that actually requires them.

**Comment by JanKallman** [comment]
Yes, I'll see if I can come up with something.

**Comment by swmal** [comment]
This will be fixed in the next version. We have a new method signature of LoadFromCollection where you can do this:

```csharp
sheet.Cells["C1"].LoadFromCollection(items, c =>
{
    c.PrintHeaders = true;
    c.HeaderParsingType = HeaderParsingTypes.Preserve;
});
```

**Comment by swmal** [comment]
Available in 5.2.1
